### ISSUE TITLE
Icon/emoji font requests

### ISSUE THREAD

**Issue by laerm0** [google/fonts#1173]
There are lots of requests for icon/emoji fonts in the old issues. [Open Sans Emoji](https://github.com/MorbZ/OpenSansEmoji) is a something for now and under Apache.

**Comment by davelab6** [comment]
Yeah, if you can list all the emoji/icon fonts that people have requested, would be great to have that tracked in this issue :)

**Comment by laerm0** [comment]
I haven't seen any requests for a specific one yet, but if I do, I'll keep track.

**Comment by Secret-chest** [comment]
Now that we have the Icons tab, we can add a new Emoji one too.
